ignition/parameters.json

```
@@ -0,0 +1,5 @@
+{
+	"F3ParametersModule": {
+		"expiration": 1753999200
```

```suggestion
		"expiration": 1754006400 /* 2025-08-01T00:00:00Z */
```

Adjusted to be Aug 1st in UTC

I don't know if we can add comments into these json files...

---

I'm not sure we can, also won't adjust it now as I want to keep the deployed state consistent for now.

---

Date is adjusted in https://github.com/filecoin-project/f3-activation-contract/pull/12/files#diff-fccf0f435c5941868521196770466bc4bdf5ff6a23afa1a6c7efc5e4c06940daR4 .  Resolving.